homework/english/BuhtiyarovP_hw_eng_2/lesson2/grammar.md

```diff
@@ -0,0 +1,38 @@
+# Упражнения Past Simple и Present Perfect
+
+### Упражнение 1.  Put the verbs in brackets in the Past Simple or in the Present Perfect.
+
+1. I (have, just) ______   a nice pot of coffee. Would you like a cup?  ---   I have just had a nice pot of coffee. Would you like a cup?
+2. I (see, not) ______   Steve this morning yet. --- I haven't seen Steve this morning yet.
+3.  Carol and I are old friends. I (know) ______   her since I (be) ______   a freshman in high school. --- Carol and I are old friends. I have known her since I were a freshman in high school.
```

Comment:
since I was

"I were" construction are used in conditional sentences. Example: "If I were you I..."